app/src/main/java/com/simplymadeapps/preferencehelper/PreferenceHelper.java

```diff
@@ -110,12 +110,17 @@ else if(Set.class.isAssignableFrom(instanceType)) {
         }
         else {
             // Store a custom non-primitive object as JSON string
+            System.out.println(new Gson().toJson(value, instanceType));
```

Comment:
Why is this here?

Reply:
I just used it for testing.  It is gone now.